ftw/upgrade/browser/templates/manage_plain.pt

```diff
@@ -163,17 +163,22 @@
                                          title="This upgrade is already installed."
                                          i18n:attributes="title" />
 
-                                    <img tal:condition="python: not upgrade['done'] and not upgrade['proposed']"
+                                    <img tal:condition="python: not upgrade['done'] and not upgrade['proposed'] and not upgrade['deferrable']"
                                          tal:attributes="src string:$portal_url/++resource++ftw.upgrade/upgrade-error.png"
                                          title="Something is wrong with this Upgrade step. This upgrade steps seems not to be installed but is not proposed. The target version may be newer than the filesystem profile version."
                                          i18n:attributes="title" />
 
+                                    <img tal:condition="upgrade/deferrable|nothing"
+                                         tal:attributes="src string:$portal_url/++resource++ftw.upgrade/upgrade-deferrable.png"
+                                         title="This upgrade is marked as deferrable."
+                                         i18n:attributes="title" />
```

Comment:
Could you fix the indenting here so that it aligns well with non-deferreable upgrades?
![42928676-46de344e-8b38-11e8-8fce-1a14e4cb055c](https://user-images.githubusercontent.com/7469/42956946-8915ad1c-8b81-11e8-95e0-d63770e608c6.png)

Reply:
Short Answer: Yes, but quick&dirty 😂 . I have set the width to 6em which leaves space for up to 3 icons. I don't think we display more than that 🤔
![screen shot 2018-07-23 at 14 19 49](https://user-images.githubusercontent.com/736583/43076144-922b44e8-8e83-11e8-8fe2-289126d4490c.png)
.

Long Answer:  This issue is not new, but also appears with other icons. I think the underlying problem is that the upgrade view tries to build a `table` with `div` and `span`. Properly aligning would make more sense with a table, if we bother we might wanna convert the listing. IMHO it is good enough with my quick&dirty fix for now.

![screen shot 2018-07-23 at 14 05 31](https://user-images.githubusercontent.com/736583/43075752-fc3cda10-8e81-11e8-8203-53f37b23ca60.png)
![screen shot 2018-07-23 at 14 07 27](https://user-images.githubusercontent.com/736583/43075754-fc5c8a5e-8e81-11e8-9b44-d186dbd48d80.png)